language/stdlib/modules/move_version.move

```diff
@@ -0,0 +1,32 @@
+address 0x0:
+
+module MoveVersion {
+    use 0x0::LibraConfig;
+    use 0x0::Transaction;
+
+    struct T {
+        major: u8,
```

Comment:
Is `u8` big enough for these?

Reply:
Agreed. I don't think there's any need to save space here, so u64 would be good even if it is total overkill.

Reply:
The reason why I put u8 over there is because the move deserializer is storing the version number as u8 and the version is represented a major and a minor number, so I'm just matching the deserializer implementation with this resource. See [here](https://github.com/libra/libra/blob/6f45031f387be5d78890cdae85f872bf54402685/language/vm/src/deserializer.rs#L109). I do agree that the choice here is totally arbitrary.

Reply:
I see. Putting 2 u64 values (major + minor) into the header of every binary Move file does seem a bit much. Depending on the answer to Sam's question about the minor version, maybe a compromise would be to use a single u64 value

Reply:
I think it's fine to have different representation for config and binary.
So if a u64 is better in Move don't worry too much